### PR TITLE
HAWQ-1377. Add more information for Ranger related GUCs in default ha…

### PR DESCRIPTION
**Pull request by wcl14**
…wq-site.xml.

### CODE REVIEW

**Comment by paul-guo-** [comment]
1. syntax error.
2. Better with a simple example?
3. Whether we could merge the three GUCs into one?
    e.g. hawq_ranger_rps_url
    $rps_server:$rps_port/$eps_suffix

**Comment by wcl14** [comment]
@paul-guo- Thanks for review. I will modify with your comments. But I don't think we should merge three GUCs into one, it follows HAWQ style.

**Comment by paul-guo-** [comment]
@wcl14  I'm not saying you do marging in this patch. I'm just wondering whether you guys working on Ranger could think whether the merging is needed. Anyway, it is just my 2 cents. Up to you guys.

File: src/backend/utils/misc/etc/hawq-site.xml [hunk 1]
```diff
@@ -149,4 +149,39 @@ under the License.
...
+        <description>Ranger plugin server address hostname.
+                     For example, HAWQ Ranger plugin server address is localhost:8432/rps.
+        </description>
+    </property>
+
+    <property>
+        <name>hawq_rps_address_port</name>
+        <value>8432</value>
+        <description>Ranger plugin server address port number.
+                     For example, HAWQ Ranger plugin server address is localhost:8432/rps.
```
Comment (ictmalili) [inline src/backend/utils/misc/etc/hawq-site.xml]: I don't think we should add three same examples for different GUCs.  They are duplicated.

File: src/backend/utils/misc/etc/hawq-site.xml [hunk 2]
```diff
@@ -149,4 +149,39 @@ under the License.
...
+        </description>
+    </property>
+
+    <property>
+        <name>hawq_rps_address_suffix</name>
+        <value>rps</value>
+        <description>Ranger plugin server suffix of restful service address.
+                     For example, HAWQ Ranger plugin server address is localhost:8432/rps.
```
Comment (ictmalili) [inline src/backend/utils/misc/etc/hawq-site.xml]: We can keep the example here. and see it co-works with the two above parameters.

File: src/backend/utils/misc/etc/hawq-site.xml [hunk 3]
```diff
@@ -163,24 +163,22 @@ under the License.
     <property>
         <name>hawq_rps_address_host</name>
         <value>localhost</value>
-        <description>Ranger plugin server address hostname.
-                     For example, HAWQ Ranger plugin server address is localhost:8432/rps.
-        </description>
+        <description>Ranger plugin server address hostname.</description>
```
Comment (linwen) [inline src/backend/utils/misc/etc/hawq-site.xml]: The description can be "The hostname of Ranger plugin server"

File: src/backend/utils/misc/etc/hawq-site.xml [hunk 4]
```diff
@@ -163,24 +163,22 @@ under the License.
...
     </property>
 
     <property>
         <name>hawq_rps_address_port</name>
         <value>8432</value>
-        <description>Ranger plugin server address port number.
-                     For example, HAWQ Ranger plugin server address is localhost:8432/rps.
-        </description>
+        <description>Ranger plugin server address port number.</description>
```
Comment (linwen) [inline src/backend/utils/misc/etc/hawq-site.xml]: The port number of Ranger Plugin Service

File: src/backend/utils/misc/etc/hawq-site.xml [hunk 5]
```diff
@@ -163,24 +163,22 @@ under the License.
...
     </property>
 
     <property>
         <name>hawq_rps_address_suffix</name>
         <value>rps</value>
-        <description>Ranger plugin server suffix of restful service address.
```
Comment (linwen) [inline src/backend/utils/misc/etc/hawq-site.xml]: the suffix of Ranger Plugin Service.

**Comment by ictmalili** [comment]
LGTM now. +1

**Pull request closed without merging**